packages/frontend/src/components/UserSettings/AccessTokensPanel/TokensTable.tsx

```diff
@@ -0,0 +1,118 @@
+import { Button, Classes, Dialog } from '@blueprintjs/core';
+import { ApiPersonalAccessTokenResponse, formatDate } from '@lightdash/common';
+import { Button as MantineButton, Paper, Table } from '@mantine/core';
+import { Dispatch, FC, SetStateAction, useEffect, useState } from 'react';
+import { useTableStyles } from '../../../hooks/styles/useTableStyles';
+import {
+    useAccessToken,
+    useDeleteAccessToken,
+} from '../../../hooks/useAccessToken';
+
+const TokenItem: FC<{
+    token: ApiPersonalAccessTokenResponse;
+    setTokenToDelete: Dispatch<
+        SetStateAction<ApiPersonalAccessTokenResponse | undefined>
+    >;
+}> = ({ token, setTokenToDelete }) => {
+    const { description, expiresAt } = token;
+
+    return (
+        <>
+            <tr>
+                <td>{description}</td>
+                <td>
+                    {expiresAt ? formatDate(expiresAt) : 'No expiration date'}
+                </td>
+                <td>
+                    <MantineButton
```

Comment:
can it just be Button? I don't think you also export blueprint button in this file

Reply:
Not yet, this PR: https://github.com/lightdash/lightdash/pull/5371 will remove the blueprint reference 😉